tests/e2e/framework/kubernetes.go

```diff
@@ -562,13 +568,30 @@ func (k *KubeInfo) Teardown() error {
 			} else {
 				istioYaml = getClusterWideInstallFile()
 			}
-
+		}
+		if *useOperator {
+			// Need an operator unique delete procedure
+			if _, err := util.Shell("kubectl -n %s get IstioControlPlane example-istiocontrolplane -o=json | jq '.metadata.finalizers = null' | kubectl -n %s apply --kubeconfig=%s -f -",
+				 "istio-operator", "istio-operator", k.KubeConfig); err != nil {
+				log.Errorf("Failed to turn off operator finalizer.")
+				return err
+			}
+			if _, err := util.Shell("kubectl delete ns %s --grace-period=0 --force --kubeconfig=%s",
+				"istio-operator", k.KubeConfig); err != nil {
+				log.Errorf("Failed to delete istio-operator namespace.")
+				return err
+			}
+			if _, err := util.Shell("kubectl delete ns %s --grace-period=0 --force --kubeconfig=%s",
+				k.Namespace, k.KubeConfig); err != nil {
+				log.Errorf("Failed to delete istio-system namespace.")
```

Comment:
the namespace in the log here should not be hardcoded?

Reply:
done.